syntax = "proto3";
package deployment;




import "flowable_sdk/model/flowable/deployment.proto";

/*
GetDeployment请求
*/
message GetDeploymentRequest {
    /*
    部署ID
    */
    string deploymentId = 1 ;
}

/*
GetDeploymentApi返回
*/
message GetDeploymentResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    flowable.FlowableDeployment data = 4 ;
}

